pkg/admin/model/override.go

```diff
@@ -0,0 +1,115 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package model
+
+import (
+	"strings"
+
+	"github.com/apache/dubbo-admin/pkg/admin/constant"
+)
+
+type Override struct {
+	Key           string           `json:"key" yaml:"key"`
+	Scope         string           `json:"scope" yaml:"scope"`
+	ConfigVersion string           `json:"configVersion" yaml:"configVersion"`
+	Enabled       bool             `json:"enabled" yaml:"enabled"`
+	Configs       []OverrideConfig `json:"configs" yaml:"configs"`
+}
+
+type OverrideConfig struct {
+	Side              string            `json:"side" yaml:"side"`
+	Addresses         []string          `json:"addresses" yaml:"addresses"`
+	ProviderAddresses []string          `json:"providerAddresses" yaml:"providerAddresses"`
+	Parameters        map[string]string `json:"parameters" yaml:"parameters"`
+	Applications      []string          `json:"applications" yaml:"applications"`
+	Services          []string          `json:"services" yaml:"services"`
+	Type              string            `json:"type" yaml:"type"`
+	Enabled           bool              `json:"enabled" yaml:"enabled"`
+	Match             ConditionMatch    `json:"match" yaml:"match"`
+}
+
+type ConditionMatch struct {
+	Address     StringMatch  `json:"address" yaml:"address"`
+	Service     StringMatch  `json:"service" yaml:"service"`
+	Application StringMatch  `json:"application" yaml:"application"`
+	Param       []ParamMatch `json:"param" yaml:"param"`
```

Comment:
Please reference the following code to define `ConditionMatch`.

https://github.com/apache/dubbo/blob/3.2/dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/configurator/parser/model/ConditionMatch.java

https://github.com/apache/dubbo/blob/3.2/dubbo-cluster/src/main/java/org/apache/dubbo/rpc/cluster/router/mesh/rule/virtualservice/match/ListStringMatch.java

Reply:
Done.